shell/browser/api/electron_api_view.cc

```diff
@@ -10,6 +10,10 @@
 #include <string>
 #include <utility>
 
+#include "ash/style/rounded_rect_cutout_path_builder.h"
+// HACK: The ash dep is not included in builds, but this file has no transitive
+// dependencies. If this ever breaks, we can copy the file into //shell/browser
+#include "ash/style/rounded_rect_cutout_path_builder.cc"
```

Comment:
yikes, any chance we can put this in chromium_src in BUILD.gn instead? #including a cc file is bad juju